src/font.c

```diff
@@ -1083,7 +1089,7 @@ gdip_create_font_from_logfont (void *hdc, void *lf, GpFont **font, BOOL ucs2)
 	result->family = NULL;
 	/* Fixme - this is wrong, but I don't know of a quick way to get the emSize */
 	result->emSize = result->sizeInPixels;
-	result->unit = UnitPixel;
+	result->unit = UnitWorld;
```

Comment:
what's the reason for this change?

Reply:
This is the default unit for a font created from a LOGFONT on windows

Reply:
ok, please mention that in the commit message.